common/src/View/RotateObjectsHandle.cpp

```diff
@@ -143,6 +143,29 @@ namespace TrenchBroom {
             }
         }
 
+        Model::Hit RotateObjectsHandle::Handle2D::pickRotateHandle(const vm::ray3& pickRay, const Renderer::Camera& camera, const HitArea area) const {
+            // Work around imprecision caused by 2D cameras being positioned at map bounds...
+            // Fixes erratic handle selection behaviour at high zoom
+            auto ray(pickRay);
```

Comment:
I'd prefer `auto ray = pickRay` or `auto ray{pickRay}`. This one gets too close to the most vexing parse for my taste.